src/api/spec/db/data/remove_duplicate_repositories_spec.rb

```diff
@@ -9,6 +9,10 @@
   let(:schema_migration) { ChangeRepositoriesRemoteProjectNameToNotNull.new }
   let(:data_migration) { RemoveDuplicateRepositories.new }
 
+  before do
+    ActiveRecord::Migration.verbose = false
```

Comment:
Doesn't this change the verbosity of all following tests (test files)?

Reply:
Then I would move this to the rspec configuration, somewhere in [support](https://github.com/openSUSE/open-build-service/tree/master/src/api/spec/support) probably, or the rails test environment.
Right now, any data migration test that runs before this one would still have the verbosity turned on.